### DISCUSSION TITLE
Any one working on Mobile Edge Computing using OMNET++ simulator?

### DISCUSSION THREAD

**Discussion by kalyan0501** [omnetpp/omnetpp#1188] [Q&A] [answered]
let me know any one doing any project work on MEC?

**Comment by giovanninardini** [comment, marked as answer]
The Simu5G library includes a model for ETSI MEC: https://simu5g.org/users-guide/mec

**Reply by kalyan0501** [reply]
Dear [giovanninardini](https://github.com/giovanninardini) what are the possible area of research in MEC?
